File: VB/Views/ItemView.vb
Imports DevExpress.XtraEditors
Imports TodoApp.ViewModels

Namespace TodoApp.Views
    Public Partial Class ItemView
        Inherits XtraUserControl
        Public Sub New()
            InitializeComponent()
            ' Initializing bindings only at runtime    
            If Not mvvmContext.IsDesignMode Then InitializeBindings()
        End Sub
        Sub InitializeBindings()
            ' Initialize the Fluent API
            Dim fluent = mvvmContext.OfType(Of ItemViewModel)()
            ' Bind the Title property to the label Text
            fluent.SetBinding(titleLabel, Function(lbl) lbl.Text, Function(x) x.Title)
            ' Bind commands to buttons
            fluent.BindCommand(btnBack, Sub(x) x.Close())
            fluent.BindCommand(btnSave, Sub(x) x.Save())
            fluent.BindCommand(btnDelete, Sub(x) x.Delete())
            ' Bind datasource to editors
            fluent.SetObjectDataSourceBinding(todoItemBindingSource, Function(x) x.Item, Sub(x) x.Update())
        End Sub
    End Class
End Namespace

File: CS/Views/ItemView.cs
namespace TodoApp.Views {
    using DevExpress.XtraEditors;
    using TodoApp.ViewModels;

    public partial class ItemView : XtraUserControl {
        public ItemView() {
            InitializeComponent();
            // Initializing bindings only at runtime    
            if(!mvvmContext.IsDesignMode)
                InitializeBindings();
        }
        void InitializeBindings() {
            // Initialize the Fluent API
            var fluent = mvvmContext.OfType<ItemViewModel>();
            // Bind the Title property to the label Text
            fluent.SetBinding(titleLabel, lbl => lbl.Text, x => x.Title);
            // Bind commands to buttons
            fluent.BindCommand(btnBack, x => x.Close);
            fluent.BindCommand(btnSave, x => x.Save);
            fluent.BindCommand(btnDelete, x => x.Delete);
            // Bind datasource to editors
            fluent.SetObjectDataSourceBinding(todoItemBindingSource, x => x.Item, x => x.Update);
        }
    }
}